.npmignore

```diff
@@ -0,0 +1 @@
+unity-tests~/
```

Comment:
Is this still useful? Seems like if the module is checked out this rule come into play.